### ISSUE TITLE
Not moving to next map

### ISSUE THREAD

**Issue by joe0x04** [packetflinger/freezetag#1]
The game just sits on the intermission screen, it never progresses to the next map.

**Comment by incith** [comment]
I just compiled and ran this - seems fine for me.

```
[2023-09-22 14:05] -------------------------------------
[2023-09-22 14:25] Timelimit hit.
[2023-09-22 14:25] Stalemate!
[2023-09-22 14:26] ------- Server Initialization -------
[2023-09-22 14:26] SpawnServer: mine3
[2023-09-22 14:26] 92 entities inhibited
[2023-09-22 14:26] 6 teams with 12 entities
[2023-09-22 14:26] -------------------------------------
[2023-09-22 14:46] Timelimit hit.
[2023-09-22 14:46] Stalemate!
[2023-09-22 14:46] ------- Server Initialization -------
[2023-09-22 14:46] SpawnServer: fact1
[2023-09-22 14:46] 34 entities inhibited
[2023-09-22 14:46] 5 teams with 10 entities
[2023-09-22 14:46] -------------------------------------
[2023-09-22 15:06] Timelimit hit.
[2023-09-22 15:06] Stalemate!
[2023-09-22 15:06] ------- Server Initialization -------
[2023-09-22 15:06] SpawnServer: base2
[2023-09-22 15:06] 55 entities inhibited
[2023-09-22 15:06] 0 teams with 0 entities
[2023-09-22 15:06] -------------------------------------
[2023-09-22 15:26] Timelimit hit.
[2023-09-22 15:26] Stalemate!
[2023-09-22 15:26] ------- Server Initialization -------
[2023-09-22 15:26] SpawnServer: mine4
[2023-09-22 15:26] 54 entities inhibited
[2023-09-22 15:26] 1 teams with 2 entities
[2023-09-22 15:26] -------------------------------------
[2023-09-22 15:46] Timelimit hit.
[2023-09-22 15:46] Stalemate!
[2023-09-22 15:46] ------- Server Initialization -------
[2023-09-22 15:46] SpawnServer: city2
[2023-09-22 15:46] 271 entities inhibited
[2023-09-22 15:46] 15 teams with 32 entities
[2023-09-22 15:46] -------------------------------------
[2023-09-22 16:06] Timelimit hit.
[2023-09-22 16:06] Stalemate!
[2023-09-22 16:06] ------- Server Initialization -------
[2023-09-22 16:06] SpawnServer: base2
[2023-09-22 16:06] 55 entities inhibited
[2023-09-22 16:06] 0 teams with 0 entities
[2023-09-22 16:06] -------------------------------------
[2023-09-22 16:26] Timelimit hit.
[2023-09-22 16:26] Stalemate!
[2023-09-22 16:27] ------- Server Initialization -------
[2023-09-22 16:27] SpawnServer: q2dm7
[2023-09-22 16:27] 0 entities inhibited
[2023-09-22 16:27] 0 teams with 0 entities
[2023-09-22 16:27] -------------------------------------
[2023-09-22 16:47] Timelimit hit.
[2023-09-22 16:47] Stalemate!
[2023-09-22 16:47] ------- Server Initialization -------
[2023-09-22 16:47] SpawnServer: waste2
[2023-09-22 16:47] 70 entities inhibited
[2023-09-22 16:47] 3 teams with 6 entities
[2023-09-22 16:47] -------------------------------------
[2023-09-22 17:07] Timelimit hit.
[2023-09-22 17:07] Stalemate!
[2023-09-22 17:07] ------- Server Initialization -------
[2023-09-22 17:07] SpawnServer: ware2
[2023-09-22 17:07] 57 entities inhibited
[2023-09-22 17:07] 6 teams with 12 entities
[2023-09-22 17:07] -------------------------------------
[2023-09-22 17:27] Timelimit hit.
[2023-09-22 17:27] Stalemate!
[2023-09-22 17:27] ------- Server Initialization -------
[2023-09-22 17:27] SpawnServer: city2
[2023-09-22 17:27] 271 entities inhibited
[2023-09-22 17:27] 15 teams with 32 entities
[2023-09-22 17:27] -------------------------------------
[2023-09-22 17:47] Timelimit hit.
[2023-09-22 17:47] Stalemate!
[2023-09-22 17:47] ------- Server Initialization -------
[2023-09-22 17:47] SpawnServer: q2dm2
[2023-09-22 17:47] 0 entities inhibited
[2023-09-22 17:47] 1 teams with 2 entities
[2023-09-22 17:47] -------------------------------------
```

**Comment by joe0x04** [comment]
Hi there!
Yes, it seems this behavior was fixed in ef8af86a9a5a45cbfa8e1be9995e695ec3e6f9b5 but this issue was never closed out.

I just checked myself and it does progress now. It will work specifying the maps via the `sv_maplist` cvar, or the `[maplist]` stanza in the _freeze.ini_ config file. 

I also noticed you were running the original single player maps in your condump. Those maps each have a "nextmap" property set on their worldspawn entity, so I'm fairly positive that progression will work even if you don't specify a map list at all.

Thanks for bringing this back to my attention! :D

**Comment by incith** [comment]
Just to clarify, I think I'm using the 'stock' maplist, that is I did not modify freeze.ini beyond the motd.

```
[maplist]
// map name, gravity level (800 normal, 100 like space), light level (1 - 10, 1 being darkest), grapple toggle
// q2dm1,0,0,1
q2dm1
q2dm2
q2dm3
q2dm5
q2dm6
q2dm7
q2dm8
base2
base3
// low gravity and no grapple
ware2,100,0,1
jail1,100
mintro
mine3
mine4,0,5
fact1
fact3
strike
power1
waste1
waste2
biggun
command
city2,800,4
boss1
###
```

It also seems to be choosing a random map from the list, which is intended as I have sv_randommap set.  Everything seems to work great.
